README.md

```diff
@@ -1 +1,33 @@
-# angular-setup-readme
+# Angular Setup + directory structure
+
+## Overview
+
+We need to ensure that we have a level-headed way of structuring our applications from the beginning, otherwise we could end up with files everywhere, which means it's hard to know exactly what you are editing and where.
+
+## Objectives
+
+- Define the core structure of an Angular application
+- Create a feature based file structure
+
+## Core Structure
+
+Angular supports modules, and our application file structure will reflect this. Each module will have directives, templates, controllers and services.
```

Comment:
What do you mean by modules? Define this more clearly. Also, we don't know what controllers, services, directives, or templates are yet. Either explain it, or let the reader know they don't have to worry about this yet.